pkgs/misc/uboot/default.nix

```diff
@@ -66,11 +67,14 @@ let
         p.setuptools # for pkg_resources
       ]))
       swig
+      which # for scripts/dtc-version.sh
     ];
     depsBuildBuild = [ buildPackages.stdenv.cc ];
 
     hardeningDisable = [ "all" ];
 
+    enableParallelBuilding = true;
```

Comment:
This will need to be verified by building **all** U-Boot derivations, but I believe it should now be working.

Reply:
I build everything we have, and it all worked.

Reply:
I don't think you built the armv6 and armv7 derivations if you only used `nixpkgs-review` :) (They'll build through `pkgsCross`.)

Reply:
ACK, on it.

Reply:
```
nix build -L -f. pkgsCross.armv7l-hf-multiplatform.ubootA20OlinuxinoLime pkgsCross.armv7l-hf-multiplatform.ubootA20OlinuxinoLime2EMMC pkgsCross.armv7l-hf-multiplatform.ubootAmx335xEVM pkgsCross.armv7l-hf-multiplatform.ubootBananaPi pkgsCross.armv7l-hf-multiplatform.ubootBananaPim3 pkgsCross.armv7l-hf-multiplatform.ubootClearfog pkgsCross.armv7l-hf-multiplatform.ubootCubieboard2 pkgsCross.armv7l-hf-multiplatform.ubootJetsonTK1 pkgsCross.armv7l-hf-multiplatform.ubootNovena pkgsCross.armv7l-hf-multiplatform.ubootOdroidXU3 pkgsCross.armv7l-hf-multiplatform.ubootOrangePiPc pkgsCross.armv7l-hf-multiplatform.ubootOrangePiZero pkgsCross.armv7l-hf-multiplatform.ubootPcduino3Nano pkgsCross.armv7l-hf-multiplatform.ubootQemuArm pkgsCross.armv7l-hf-multiplatform.ubootRaspberryPi2 pkgsCross.armv7l-hf-multiplatform.ubootRaspberryPi3_32bit pkgsCross.armv7l-hf-multiplatform.ubootRaspberryPi4_32bit pkgsCross.armv7l-hf-multiplatform.ubootUtilite pkgsCross.armv7l-hf-multiplatform.ubootWandboard
```

Everything there built AOK :)